pkg/images/images.go

```diff
@@ -65,3 +73,28 @@ func Pull(dir, name, image string) error {
 
 	return nil
 }
+
+func checkPreloadedImages(dir string) (bool, error) {
+	_, err := os.Stat(dir)
+	if os.IsNotExist(err) {
+		return false, nil
+	} else if err != nil {
+		logrus.Errorf("Unable to find images in %s: %v", dir, err)
```

Comment:
Error strings shouldn't be capitalize but should follow repo convention. https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#error-strings